src/_xoConnection.js

```diff
@@ -114,6 +115,12 @@ class XoConnection extends Xo {
     return job;
   }
 
+  async createDurableResources() {
+    const { id } = await this.call("remote.create", config.remotes.default);
```

Comment:
This helper should be generic.

Reply:
```javascript
// _xoConnection.js
  async createDurableResources() {
    const resources = {}
    for (let i = 0; i < config.durableResources.length; i++) {
      const durableResources = config.durableResources[i]
      for (const typeOfResource in durableResources) {
        for (const resource in durableResources[typeOfResource]) {
          const { create, delete: deleteMethod, deleteParams, params } = durableResources[typeOfResource][resource];
          const { id } = await this.call(create, params);
          this._durableResourceDisposers.push(deleteMethod, { id, ...deleteParams });
          resources[typeOfResource] = { ...resources[typeOfResource], [resource]: id };
        }
      }
    }
    return resources;
  }
```
```toml
[[durableResources]]
  [durableResources.remotes]
    default = { create = 'remote.create', params = { name = 'default_remote', url = 'nfs://192.168.100.5:/tank' }, delete = 'remote.delete', deleteParams = {} }
```
Is that what you mean in this comment and the previous one?

Reply:
```toml
[preCreatedResources]
  [preCreatedResources.remotes]
    default= { name = '', url = '' }
```
As output we should have:
```json
resources: {
  "remotes": {
    "default": id
  }
}
```

Reply:
The result of this solution is already : 
```javascript
resources: {
  remotes: {
    default: id
  }
}
```
I will change `durableResources` -> `preCreatedResources`.

Reply:
The result of this solution is already : 
```javascript
resources: {
  remotes: {
    default: id
  }
}
```
I will change `[[durableResources]]` -> `[preCreatedResources]` :+1: .

Reply:
And can I keep the same method name (`createDurableResources()`) ? Or Is it still not enough significant ?